uniconnect-epfl/uniconnect
# feat: interests can now be selected when creating an event

# What I did
Allow to select interests when creating an event or announcement.

# How I did it
Reused `InterestsScreen` from the registration screen by making it more generic using `useRoute` params (`eventMode`).

# How to verify it
1. Go to event screen/announcement screen. 
2. Create an event/announcement.
3. Select some interests that should be added.
4. Publish the event/announcement

# Demo video
https://github.com/uniconnect-epfl/uniconnect/assets/11707683/2f781e79-804e-42ac-a137-5ee0a8408755

# Pre-merge checklist
The changes I have introduced:
